bigtable-test/bigtable-emulator-maven-plugin/README.md

```diff
@@ -66,4 +67,47 @@ Usage:
       Connection connection = BigtableConfiguration.connect("fakeproject", "fakeinstace");
    ```
 - If `maven.test.skip` is set to true the emulator will not start. The emulator can also be controlled directly 
-  by setting the boolean property `bigtable.emulator.skip`.
\ No newline at end of file
+  by setting the boolean property `bigtable.emulator.skip`.
+
+- While using Docker, You can use container's `host` and `port` details for emulator:
+  ```java
+  GenericContainer container = // Container Contorller
+
+  Configuration config = BigtableConfiguration.configure("fakeproject", "fakeinstance");
+  config.set(container.getContainerIpAddress() + ":" + container.getMappedPort(TEST_HTTP_PORT));
+  Connection connection = BigtableConfiguration.connect(config);
+  ```
+
+
+## [Google-cloud-Bigtable-emulator](https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-testing/google-cloud-bigtable-emulator)
+
+  This is the latest bigtable emulator, It includes below features over `bigtable-emulator-maven-plugin`:
```

Comment:
use a semi colon, or a full stop. If semi colon, do not start with capital "I". If you are using a full stop, then you can start with a capital letter.